src/datascience-ui/ipywidgets/renderer/index.ts

```diff
@@ -27,3 +32,20 @@ export const activate: ActivationFunction = (_context) => {
         }
     };
 };
+
+function hookupTestScripts(context: RendererContext<unknown>) {
```

Comment:
If the global entry point exists, then we setup the hooks, else we don't care.
This global object will be available only in tests